src/System.Globalization.Extensions/tests/Normalization/StringNormalizationTests.cs

```diff
@@ -31,7 +31,7 @@ public void IsNormalized_Invalid()
             Assert.Throws<ArgumentException>("strInput", () => "\uFFFE".IsNormalized()); // Invalid codepoint
             Assert.Throws<ArgumentException>("strInput", () => "\uD800\uD800".IsNormalized()); // Invalid surrogate pair
 
-            Assert.Throws<ArgumentNullException>("strInput", () => StringNormalizationExtensions.IsNormalized(null));
+            Assert.Throws<NullReferenceException>(() => StringNormalizationExtensions.IsNormalized(null));
```

Comment:
I believe that the general convention for extension methods has been to explicitly check for null.